rules/sources/online_identifiers.yaml

```diff
@@ -15,7 +15,7 @@ sources:
     isSensitive: False
     sensitivity: medium
     patterns:
-      - "(?i).*(device[^\\s/(;)#|,=!>]{0,5}(?:id|info|details|token))|imei"
+      - "(?i).*(device[^\\s/(;)#|,=!>]{0,5}(?:id|info|details|token))|imei|fingerprint"
```

Comment:
I guess `fingerprint` was there earlier, but was resulting in FP, @HiteshMah-Jan can confirm

Reply:
I can remove this if thats the case. @HiteshMah-Jan @khemrajrathore

Reply:
Let me know if I should remove this.